docs/conf.py

```diff
@@ -80,7 +80,7 @@
     import sphinx_rtd_theme
 
     html_theme = "sphinx_rtd_theme"
-    html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
+##    html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
```

Comment:
Someone once said to me that including unrelated "fixes" in a PR is bad practice...

Reply:
And you're absolutely correct 🤦‍♂️